api/config/config_perf.json

```diff
@@ -16,7 +16,8 @@
     "runtimeImages": {
       "gce": [],
       "dataproc": []
-    }
+    },
+    "gceVmZone": "us-central1-a"
```

Comment:
perf I believe runs in the test.firecloud.org GCP org as well. May make sense to use us-central1-d here too.

Reply:
done.  (if we face the same issue again, we might consider to make perf use different zone as test)